app.py

```diff
@@ -163,11 +163,191 @@ def signUp():
     del result['access_token']
     del result['verification_token']
 
+    db.session.close()
+
     # Return the user
     return {
-        "user": result
+        "user": session['access_token']
+    }
+
+@app.route('/approveClient', methods=['PUT'])
+@http_guard(renew=True, nullable=False)
+def approveClient(token_claims):
+    body = request.get_json(force=True)
+    # Check that the role of the requestee is COACH
+    if token_claims['role'] != Role.COACH.name:
+        return {
+            "error": "Expected role of COACH"
+    }, 400
```

Comment:
Just some indentation thing
```suggestion
        }, 400
```